edgelet/contrib/config/linux/template.toml

```diff
@@ -0,0 +1,154 @@
+# ==============================================================================
+# Hostname
+# ==============================================================================
+#
+# Comment out the next line to override the default hostname of this device.
+#
+# hostname = "my-device"
+
+
+# ==============================================================================
+# Provisioning
+# ==============================================================================
+#
+# Comment out one block and replace the stub values with yours.
+
+
+## Manual provisioning with connection string
+#
+# [provisioning]
+# source = "manual"
+# connection_string = "HostName=example.azure-devices.net;DeviceId=my-device;SharedAccessKey=YXppb3QtaWRlbnRpdHktc2VydmljZXxhemlvdC1pZGU="
+
+
+## Manual provisioning with symmetric key
+# [provisioning]
+# source = "manual"
+# iothub_hostname = "example.azure-devices.net"
+# device_id = "my-device"
+# 
+# [provisioning.authentication]
+# method = "sas"
+#
+# device_id_pk = { value = "YXppb3QtaWRlbnRpdHktc2VydmljZXxhemlvdC1pZGU=" }     # inline key (base64), or...
```

Comment:
Again - remove the actual key. 
But in general, we also need documentation or comments on each of these.